lineapy/graph_reader/artifact_collection.py

```diff
@@ -338,16 +357,40 @@ def _compose_module(
                 if len(line.lstrip(" ").rstrip(",")) > 0
             ]
 
+        module_input_parameters = [
+            InputVariable(
+                variable_name=x.split("=")[0].strip(" "),
+                value=x.split("=")[1].strip(" "),
+            )
+            for x in input_parameters_body
+        ]
+
         module_input_parameters_body = (
-            ",".join(input_parameters_body)
-            if len(input_parameters_body) <= 1
-            else "\n"
-            + f",\n{indentation_block}".join(input_parameters_body)
+            ""
+            if len(module_input_parameters) == 0
+            else module_input_parameters[0].default_args
+            if len(module_input_parameters) == 1
+            else f"\n{indentation_block}"
+            + f",\n{indentation_block}".join(
+                [param.default_args for param in module_input_parameters]
+            )
+            + ",\n"
```

Comment:
this is becoming confusing now, instead of putting this in bracket like here, split into multiple if-else is suggested.